### DISCUSSION TITLE
Is it possible to use a CloudSQL sidecar?

### DISCUSSION THREAD

**Discussion by djMax** [alexandrevilain/temporal-operator#599] [Q&A] [answered]
We are running in GCP (GKE) using a CloudSQL postgres instance. I have added the cloudsql sidecare to the deployment overrides, but it seems like the operator spins up a bunch of jobs first, and those jobs do not inherit the sidecar options. Is it possible to use the operator to deploy in a GCP CloudSQL environment?

**Comment by alexandrevilain** [comment, marked as answer]
Hi! 

Thanks for exposing your use-case.
This is unfortunately not supported by the operator for now. 

This feature could be a great addition to the operator, let's creeate an issue from this conversation to track this.

**Reply by alexandrevilain** [reply]
Issue created: https://github.com/alexandrevilain/temporal-operator/issues/600
